go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/ReneKroon/ttlcache/v2 v2.3.0
 	github.com/aws/aws-sdk-go v1.37.30
-	github.com/c-bata/go-prompt v0.2.5
+	github.com/c-bata/go-prompt v0.2.6
```

Comment:
go-prompt update is necessary because it fixes other build errors: https://github.com/c-bata/go-prompt/pull/224.

I will take a look at Windows implementation for error handling.

Reply:
Ah okay. Will try fixing it